TEST.md

```diff
@@ -94,110 +94,93 @@ bin/shellcheck -x bin/*
 The `test/` directory contains a test suite that can be run to validate Linkerd
 functionality via a series of end-to-end tests.
 
-### Prerequisites
+### Prerequisites for default behavior
 
-The integration test suite operates on your currently configured Kubernetes
-cluster. Prior to running the test suite, verify that:
+The integration tests will configure their own KinD clusters by default. There
```

Comment:
isn't `kind` a prerequisite?  What about resource requirements?  Is `kind` running on docker-for-desktop on a 4 year old macbook going to cut it?

Reply:
> isn't kind a prerequisite? What about resource requirements?

The script uses `bind/kind` which downloads `kind` if it is not present. You don't need `kind` in your `PATH` to use the script.

> Is kind running on docker-for-desktop on a 4 year old macbook going to cut it?

I haven't had any issues with running `kind` locally on my 2015 macbook. If anything, Docker For Mac with Kubernetes turned on seems to use a lot more resources than `kind` ever does; I rarely use Docker For Mac these days because of that.

Also keep in mind we already use `kind` on the GitHub Action runners which I think are 2-core machines with 7 GB of ram. Knowing we don't have issues on those, I think we're good to assume running this locally will be fine as well.